hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/CarbonitingMetadataFreezerActor.scala

```diff
@@ -0,0 +1,124 @@
+package cromwell.services.metadata.hybridcarbonite
+
+import java.nio.file.StandardOpenOption
+
+import akka.actor.{ActorRef, FSM, LoggingFSM}
+import cats.data.NonEmptyList
+import cromwell.core.WorkflowId
+import cromwell.core.io.{AsyncIo, DefaultIoCommandBuilder}
+import cromwell.services.MetadataServicesStore
+import cromwell.services.metadata.MetadataArchiveStatus.{ArchiveFailed, Archived}
+import cromwell.services.metadata.MetadataService.GetMetadataAction
+import cromwell.services.metadata.hybridcarbonite.CarbonitingMetadataFreezerActor._
+import cromwell.services.metadata.impl.MetadataDatabaseAccess
+import cromwell.services.metadata.impl.builder.MetadataBuilderActor.{BuiltMetadataResponse, FailedMetadataResponse}
+import cromwell.services.metadata.{MetadataArchiveStatus, MetadataQuery}
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try}
+
+final class CarbonitingMetadataFreezerActor(carboniterConfig: HybridCarboniteConfig,
```

Comment:
isn't carboniting and freezing the same thing in this context? seems redundant.

Reply:
My thinking was that this is the `FreezerActor` of the `CarbonitingMetadata` family of actors. Alternative suggestions welcome though